hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/validation/AttributeValidatorImpl.java

```diff
@@ -91,7 +91,7 @@ public void validateExpiry(long expiry) {
                 context.configuration().getConfigData(EntitiesConfig.class).maxLifetime();
         final var now = context.consensusNow().getEpochSecond();
         final var expiryGivenMaxLifetime = now + maxEntityLifetime;
-        validateTrue(expiry > now && expiry <= expiryGivenMaxLifetime, INVALID_EXPIRATION_TIME);
+        validateTrue(expiry >= now && expiry <= expiryGivenMaxLifetime, INVALID_EXPIRATION_TIME);
```

Comment:
Why would we want to allow an expiry time of now?

Reply:
In TokenCreateHandler, when calls context.expiryValidator().resolveCreationAttempt(false, givenExpiryMeta); which calls context.attributeValidator().validateExpiry(effectiveExpiry); This effectiveExpiry is context.consensusNow().getEpochSecond() (=now) if you don't have autoRenewAccount. If you don't add now check here, you will get INVALID_EXPIRATION_TIME, test cannotDeleteTreasuryAccount will fail.

Reply:
I agree with @mhess-swl. It is kind of weird to modify this check, which already exists in mono. Can we be sure that none of the dozens of other places that call this method are affected by this change? Is it possible to fix this in `TokenCreateHandler`?

Reply:
Removed the code change to TokenCreateHandler.